core/modules/proto/replication.proto

```diff
@@ -0,0 +1,514 @@
+/*
+ * LSST Data Management System
+ * Copyright 2011-2017 LSST Corporation.
```

Comment:
And it's not LSST Corp either (at least not since 2015). I think you want something like:

> Copyright 2018 The Board of Trustees of the Leland Stanford Junior University, through SLAC National Accelerator Laboratory

See https://developer.lsst.io/stack/package-docs.html?highlight=stanford#standard-institutional-copyrights

Reply:
Hi Tim,
if that's true then every single file in the Qserv package has this problem (or "problem"). Just check the code to see if  I'm right. There are ~600 files which have either:
  Copyright 2011-2017 LSST Corporation.
or
  Copyright 2015 AURA/LSST.
So, I would rather leave this 'Copyright' 'problem' till some global refactoring (code cleanup) of the whole package.

Reply:
The current policy is not to put copyright in the files at all. I don't think there is a plan for global refactoring so I'd rather be right in one place than wrong everywhere. One option is to create the `COPYRIGHT` file and put the correct copyright in that (2011-2015 LSST Corp, 2015-2018 SLAC) and leave the brokenness in the file for now. Maybe @ktlim or @fritzm can say more.

Reply:
Note that GPL rules require us to put the GPL short version in each file so there is still a need for it (although they don't say it has to be at the start of the file).

Reply:
Added the COPYRIGHT file with proposed content as per your suggestion